### DISCUSSION TITLE
Welcome to DINO-X-API Discussions!

### DISCUSSION THREAD

**Discussion by rentainhe** [IDEA-Research/DINO-X-API#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guozhimeng-0815** [comment]
我在使用https://cloud.deepdataspace.com/playground/dino-x时，确保在已经登录且图片合法的情况下，试图上传我的图片，一直上传失败之后回到登录界面，这是怎么回事
